packages/atomic/src/components/search/atomic-rating-range-facet/e2e/atomic-rating-range-facet.e2e.ts

```diff
@@ -0,0 +1,49 @@
+import {expect, test} from './fixture';
```

Comment:
If you want you can look at what I did in atomic-color-facet for stories / e2e. I think the pattern is good (we mock network requests and test the happy path).

https://github.com/coveo/ui-kit/pull/6498/files#diff-f25e96b83e27fc521347ffbbba8b31f863078e83dad96f4135b5fac11fa32644